wrappers/azurekeyvault/options.go

```diff
@@ -110,6 +112,9 @@ type options struct {
 	withResource        string
 	withVaultName       string
 	withKeyName         string
+	withAuthMethod      string
+	withX509Cert        string
+	w
```

Comment:
Typo perhaps? x509 cert support looks incomplete?

Reply:
yes sure, its only a draft..., will fix it